_posts/2016-06-15-beyond-progressive-web-apps-part-2.md

```diff
@@ -0,0 +1,176 @@
+---
+
+title: 'Beyond Progressive Web Apps Part 2: We Need to Know What’s New'
+author: Jan
+layout: blog-post
+comments: true
+
+---
+
+This is part two of a three part series about data sync beyond Progressive Web Apps. [Please read part one, before continuing](http://hood.ie/blog/beyond-progressive-web-apps-part-1.html "Beyond Progressive Web Apps Part 1").
+
+Say we have a news app that runs on mobile devices and a server that publishes new stories. Blogs and [RSS](https://en.wikipedia.org/wiki/RSS) might be a good real-world example.
```

Comment:
change 'might be' to 'are'. makes you sound more confident in what you're saying (and I know you are)